test/input-range/input-range.spec.jsx

```diff
@@ -255,6 +255,30 @@ describe('InputRange', () => {
     component.detach();
   });
 
+  it('notifies the parent component when dragging starts', () => {
+    const onChange = jasmine.createSpy('onChange').and.callFake(value => component.setProps({ value }));
+    const onChangeStart = jasmine.createSpy('onChangeStart');
+    const jsx = (
+  	<InputRange
+  	  maxValue={20}
+  	  minValue={0}
+  	  value={{ min: 2, max: 10 }}
+  	  onChange={onChange}
+  	  onChangeStart={onChangeStart} />
+  	);
+    const component = mount(jsx, { attachTo: container });
+    // eslint-disable-next-line quotes
```

Comment:
Instead of disabling this rule, can you actually fix the ESLint issue? So, just use single quote `'Slider [onMouseDown]'`.